web/src/components/filings-calendar/FilingsCalendarTaxAccess.test.tsx

```diff
@@ -188,6 +188,45 @@ describe("<FilingsCalendarTaxAccess />", () => {
   it("displays alert on success", async () => {
     mockApi.postTaxFilingsOnboarding.mockResolvedValue({
       ...userDataWithPrefilledFields,
+      profileData: {
+        ...userDataWithPrefilledFields.profileData,
+        naicsCode: "123456",
+        municipality: {
+          name: "Absecon",
+          county: "",
+          id: "",
+          displayName: "",
```

Comment:
Why do we need to add this new data to the mock? I don't see anything in the expectations that is expecting the municipality or naics code to appear on the page

Reply:
same question for the test below